### ISSUE TITLE
1% reject -is normal?

### ISSUE THREAD

**Issue by Moskga** [bzminer/bzminer#9]
bzminer 7.1.3  metapool.

5*1070, win7 drivers 472.12  with and without overclock - got reject

**Comment by cloudis31** [comment]
Yes, this is normal, those are stales, any network delay can cause the share to get to the pool a little late

**Comment by Moskga** [comment]
<Warn> GPU: 4:0 solution rejected:  error code: 26 invalid block chain index

why?

**Comment by bzminer** [comment]
yeah, i believe these are just stale, it wasn't able to find  the chain because the share was too late

**Comment by mtmtmt123** [comment]
in 40min i have 37 rejected, on official miner is this not (alph)

**Comment by bzminer** [comment]
@mtmtmt123 how many accepted do you have? its not uncommon to have 1-2% stale shares (they show as rejected)

**Comment by mtmtmt123** [comment]
Alephium Flight Sheet
bzminer-ver. 7.1.3 v7.1.3
97.19%A 1558R 45

**Comment by cloudis31** [comment]
@mtmtmt123 that could be, stales are very sensitive to high ping. An optimization I'm working on is improving the time between when work is received to time share is submitted, I believe there is a little room for improvement there

**Comment by Moskga** [comment]
> yeah, i believe these are just stale, it wasn't able to find the chain because the share was too late

No this error only on 7.1.3, 10% оf all shars!!!
7.1.12B more stabile !!! ( overclock some)

**Comment by bzminer** [comment]
@Moskga you should get the latest version, v7.1.5, it has a bug fix in the mining algorithm that cause roughly 10% invalid shares for some users (due to an optimization that i have disabled for now, will look back into re-enabling that optimization after i can find out why its causing invalid shares sometimes)

**Comment by t72o** [comment]
I've got many rejects in solo mining
 Found a blake3 solution! (alphstratum pool may not respond)
 Found a blake3 solution! (alphstratum pool may not respond)
 Found a blake3 solution! (alphstratum pool may not respond)
 Found a blake3 solution! (alphstratum pool may not respond)
 <Warn> GPU: 40:0 solution rejected:  (1ms)

 <Warn> GPU: 40:0 solution rejected:  (1ms)

 <Warn> GPU: 40:0 solution rejected:  (0ms)

 <Warn> GPU: 40:0 solution rejected:  (1ms)

Also there is accepted ones:
 Found a blake3 solution! (alphstratum pool may not respond)
 Pool accepted alph solution from GPU 40:0! (3ms)

1d 2h 5m 50s | 3/16/-/- | 1h  | 7mh/w | 28.23gh | 3.26gh

3 accepted and 16 rejected on 1day uptime, is this normal?
